EFCore.CheckConstraints/EFCore.CheckConstraints.csproj

```diff
@@ -19,6 +19,10 @@
     <RepositoryUrl>git://github.com/efcore/EFCore.CheckConstraints</RepositoryUrl>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Platform)'=='AnyCPU'">
```

Comment:
This seems like an accidental addition that should be removed.

Reply:
That group of lines in the project enables the XML documentation comments:

```xml
<PropertyGroup Condition="'$(Platform)'=='AnyCPU'">
  <DocumentationFile>obj\EFCore.CheckConstraints.xml</DocumentationFile>
</PropertyGroup>
```

Reply:
Please add `<GenerateDocumentationFile>true</GenerateDocumentationFile>` instead in the global properties section of the csproj, [as documented here](https://docs.microsoft.com/en-us/dotnet/csharp/codedoc).

Reply:
Done. (Commit postponed until comments have all been resolved.)

Looking forward to your suggestions on my reply to your other comment.